src/galileo/resources/models/http_validation_error.py

```diff
@@ -45,10 +45,12 @@ def from_dict(cls: type[T], src_dict: dict[str, Any]) -> T:
         d = src_dict.copy()
         detail = []
         _detail = d.pop("detail", UNSET)
-        for detail_item_data in _detail or []:
```

Comment:
this line was incorrect, cause `for x in "asdda"` - also will work, which we don't want.